frontend/components/bc/BcTranslation.vue

```diff
@@ -0,0 +1,58 @@
+<script lang="ts" setup>
+import type { MessageSchema } from '~/i18n.config';
+import type { KeyPaths } from '~/types/helper';
+
+const props = defineProps<{
+  /**
+   * The path to the key in the translation file (e.g. en.json)
+   */
+  keypath: KeyPaths<MessageSchema>,
+  linkpath?: KeyPaths<MessageSchema>,
+  tag?: keyof HTMLElementTagNameMap
+   /**
+   * URL to link to
+   * 
+   * @example
+   * 
+   * Translation key has to be under `${keypath}._link`
+   * 
+   *  // en.json
+   * {
+   *  "notifications": {
+   *   "template": "For further information {_link}"
+   *   "_link": "Click here"
+   * }
+   */
+  to?: string,
+}>()
+
+const isExternal = computed(() => {
+  return !props?.to?.startsWith('/')
```

Comment:
not sure, but wouldn't `props?.to?.startsWith('http')` be the saver choice or is there a reason you want it to be external if no to is provided?

Reply:
(as I always receive emails since the day I commented, I saw this)

Maybe you want to have a look at [this way](https://github.com/gobitfly/beaconchain/blob/67346b238b1ad34b9d4b1f8613106ff2180cd73b/frontend/components/bc/MiniParser.vue#L159) to determine automatically the target of a link. I think it catches more cases than just testing for `/`.

Reply:
@thib-wien `helpful input` is always appreciated.

I think we could remove it entirely. Had another look at the [docs](https://nuxt.com/docs/api/components/nuxt-link#handling-404s).
- it should be applied automatically for `absolute urls` and when `target` is added
- it should only be necessary if we would have other `apps` running on the same domain or if we link to stuff in `/public`

Reply:
Ah, we were talking about two different things :)